docs/en/operation/Troubleshooting.md

```diff
@@ -269,6 +269,70 @@ are
 - `ADVANCED`: samples and tracks recent accesses, higher overhead
 - `PARANOID`: tracks for leaks on every resource allocation, highest overhead. 
 
+## Master Internal Monitoring
+
+Alluxio master periodically checks its resource usage, including CPU and memory usage, and several internal data structures 
+that are performance critical. This interval is configured by `alluxio.master.throttle.heartbeat.interval` (defaults to 3 seconds).
+On every sampling point in time (PIT), Alluxio master takes a snapshot of its resource usage. And a certain number of PIT snapshots
+(`alluxio.master.throttle.observed.pit.number`, defaults to 3) are recorded, those will be used to generate an aggregated
+one in a sliding window. 
+
+Each PIT includes the following metrics.
+
+```console
+directMemUsed=5268082, heapMax=59846950912, heapUsed=53165684872, cpuLoad=0.4453061982287778, pitTotalJVMPauseTimeMS=190107, totalJVMPauseTimeMS=0, rpcQueueSize=0, pitTimeMS=1665995384998}
+```
+
+- `directMemUsed`: direct memory allocated by ByteBuffer.allocateDirect
+- `heapMax` : the allowed max heap size
+- `heapUsed` : the heap memory used
+- `cpuLoad` : the cpu load
+- `pitTotalJVMPauseTimeMS` : aggregated total JVM pause time from the beginning
```

Comment:
？？ it's a PIT, but it's a aggregated time from the beginning (what beginning? the process start time?)

Reply:
same data structure, not used in aggregated env.

Reply:
if the aggr one is based on pit2 (pit2 - pit1), pit3(pit3 - pit2), pit4 (pit4 - pit3), then this number would be (pit4 - pit1)

Reply:
If this is already aggregated, then it shouldn't be included in a PIT value, but included in an aggregated representation of several PITs. Listing it here in the description of a PIT is confusing.